Rover/GCS_Mavlink.cpp

```diff
@@ -845,6 +845,11 @@ void GCS_MAVLINK_Rover::handle_set_attitude_target(const mavlink_message_t &msg)
     }
 }
 
+void GCS_MAVLINK_Rover::send_acc_ignore_must_be_set_message(const char *msgname)
```

Comment:
sorry to be annoying but can you add a comment above the method?  Just copying the method from the .h is fine.